addons/skin.estuary/1080i/DialogContextMenu.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <window>
-	<defaultcontrol>1000</defaultcontrol>
+	<defaultcontrol>996</defaultcontrol>
```

Comment:
The "real" buttons start with 1001 (see https://github.com/xbmc/xbmc/blob/8d4a5bba55638dfd0bdc5e7de34f3e5293f99933/xbmc/dialogs/GUIDialogContextMenu.cpp#L53 ), so i think 1000 doesnt make too much sense here.